OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseDetailFragment.java

```diff
@@ -303,6 +303,10 @@ private void configureEnrollButton() {
 
         if (mEnrolled) {
             mEnrollButton.setText(R.string.view_course_button_text);
+        } else if (courseDetail.invitation_only) {
+            mEnrollButton.setText(R.string.invitation_only_button_text);
+            mEnrollButton.setBackgroundColor(getResources().getColor(R.color.edx_brand_gray_back));
```

Comment:
This background color doesn't pass the WCAG AA standard for foreground-background contrast ratio.

p.s. contrast ratio can be checked here: http://webaim.org/resources/contrastchecker/
p.p.s. `edx_brand_gray_base` seems like the best background color in this case.